Resources/Server Info/Park_Rules.txt

```diff
@@ -31,34 +39,34 @@
 [color=#ff0000]IN GAME & ROLEPLAY RULES[/color]
 
 [color=#a4885c]9.[/color] Pick a realistic name that could appear on a birth certificate with at least a first and last name (leeway is given to this for Clowns, Mimes, and non-human races).
-    - Names of notable famous or fictional persons or names that resemble/parody them are strictly forbidden. You are not clever if you slightly change a famous name around. Terrible names open you up to being politely reminded to change it, smited, or instantly banned, depending on severity.
+    - Names of notable famous or fictional persons or names that resemble/parody them are strictly forbidden. You are not clever if you slightly change a famous name around. Terrible names open you up to being politely reminded to change it, smote, or instantly banned, depending on severity.
```

Comment:
technically the correct past participle here is "smitten" but given the more common usage of that word is an adjective meaning struck by affection or longing, it may be better to reword this entirely or use "smited" since it sounds more correct while retaining unambiguity

Reply:
who are you

Reply:
i ask myself this constantly

Reply:
ok, I'll rewrite it for clarity since I dislike the alternate forms of 'smite'